src/components/MovieDetails/MovieDetails.js

```diff
@@ -2,21 +2,16 @@ import React, { Component } from 'react';
 import { getRatings } from '../../actions';
 import { connect } from 'react-redux';
 
-import { apiFetchRatings, apiPostNewRating } from '../../apiCalls/apiCalls';
+import { apiFetchRatings, apiPostNewRating, apiDeleteRating } from '../../apiCalls/apiCalls';
 
 class MovieDetails extends Component {
   constructor() {
     super();
     this.state = {
-      userRating: null
+      userRating: 1
     }
```

Comment:
It seems odd to set this to 1 by default? Should it be set to 0 or '' by default? That would allows us to handle errors on this as well.

Reply:
I think the lowest value a user can rate is 1, so 0 will not work in this case.

Reply:
But if we set it to 0 by default and the user does not select a rating before clicking submit, then we can use that value of 0 to know that they haven't selected a rating and display an error

Reply:
Agree, I understand the logic behind not being able to have it be null, but I don't think we want it to default to 1.

Reply:
When a page loads, the start value is at 1, so that is why I made the default value at 1.  Unless the start value changes, then the userRating default value should match the starting value.

Reply:
Maybe we just need to put a default value on the select form?

Reply:
Like so: `<option value="default">-- Select a Rating --</option>`. This way we can handle the case in which a user does not select a rating. So the logic could be like: if select value is anything except 'default' then we can submit the form.

Reply:
Did you test this in the browser to make sure that the "You haven't rated this movie yet!" message still shows up

Reply:
That's a good way of doing it!